.github/workflows/pr-ci.yml

```diff
@@ -86,13 +87,85 @@ jobs:
       checkout-ref: ${{ needs.checkout-branch.outputs.PR_head_ref  }}
       skip-itests: ${{ matrix.arch != 'amd64' }}
 
+# Add a new step to use /retest
+  retest-integration:
+    needs: [code-analysis, checkout-branch]
+    if: contains(github.event.comment.body, '/retest')
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    uses: ./.github/workflows/ci-build-image.yml
+    with:
+      build-arch: ${{ matrix.arch }}
+      run-tests-only: true
+      pr-number: ${{ github.event.issue.number }}
+      sha-value: ${{ needs.checkout-branch.outputs.PR_head_sha }}
+
+  start-comment:
+    runs-on: ubuntu-latest
+    needs: [check-before-build]
+    if: ${{ needs.check-before-build.result == 'success' }}
```

Comment:
don't need this if statement- the `needs: [check-before-build]` ensures that this job only runs if the `check-before-build` job succeeded

Reply:
same with [L126](https://github.com/aali309/cryostat/blob/e474dae3a41e9da2ee214aaa1af3388ffc6b5a6c/.github/workflows/pr-ci.yml#L126) and [L145](https://github.com/aali309/cryostat/blob/e474dae3a41e9da2ee214aaa1af3388ffc6b5a6c/.github/workflows/pr-ci.yml#L145C1-L145C1)

Reply:
> don't need this if statement- the `needs: [check-before-build]` ensures that this job only runs if the `check-before-build` job succeeded

Yes that is the intention, it should run only when the jobs mentioned succeed

Reply:
yes